bin/configs/kotlin-server-javalin.yaml

```diff
@@ -0,0 +1,7 @@
+generatorName: kotlin-server
+outputDir: samples/server/petstore/kotlin-server/javalin
```

Comment:
can you please update the kotlin server github workflow with the new sample directory so that CI can test it moving forward?
```
.github/workflows/samples-kotlin-server-jdk17.yaml 
.github/workflows/samples-kotlin-server.yaml
```

Reply:
Thanks for the suggestion. Done! ✅ 

It looks like one of the Spring Boot-related tests are now failing. That example is already broken on the `master` branch:

https://github.com/OpenAPITools/openapi-generator/blob/36e4e4f3167707b3a657121897fdf6cfc33b85f1/samples/server/petstore/kotlin-springboot-request/src/main/kotlin/org/openapitools/api/StoreApi.kt#L74

Please let me know if there's anything else I can do within the scope of this MR 😊 thanks!

Reply:
agreed it's not related to this change